
builtin_interfaces/Time stamp
float32 angular_rate_radians_per_second

# TOPICS offboard/controller_input_angular_rate 
